src/nft/utils/buildSellObject.ts

```diff
@@ -0,0 +1,16 @@
+export const buildSellObject = (amount: string) => {
+  return {
+    address: '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
```

Comment:
What is this address?

Reply:
It's effectively the same as the null address and it's what the genieswap contract uses for buying with ETH. @azflin might be able to discuss this further if you have more questions on it.